src/qt/peertablemodel.h

```diff
@@ -90,6 +91,9 @@ public Q_SLOTS:
         tr("Direction"),
         /*: Title of Peers Table column which describes the type of
             peer connection. The "type" describes why the connection exists. */
+        tr("Duration"),
+        /*: Title of Peers Table column which describes the duration of
```

Comment:
You added this translator comment in reverse, the comment on lines 95 and 96 should be above line 94 in this diff.

Reply:
Thanks for your attention on this.